package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "todolist",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/binodbhusal/ToDoList.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/binodbhusal/ToDoList/issues"
+  },
+  "homepage": "https://github.com/binodbhusal/ToDoList#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.7.3",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^7.1.8",
+    "style-loader": "^3.3.2",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack": "^5.81.0",
+    "webpack-cli": "^5.0.2"
+  }
+}
```

Comment:
- [ ] Webpack dev server is not properly configured. Kindly ensure that basic commands for building Webpack in development mode, production mode is configured by adding the necessary script commands.

**Error starting Webpack dev server:**
![image](https://user-images.githubusercontent.com/12745474/235915302-26f3e7de-2b01-4b3b-84df-2b76d68ef61b.png)